src/structures/Guild.js

```diff
@@ -688,6 +699,15 @@ class Guild extends Base {
       );
   }
 
+  /**
+   * Fetches the welcome screen for this guild.
+   * @returns {Promise<WelcomeScreen>}
+   */
+  async fetchWelcomeScreen() {
+    const data = await this.client.api.guilds(this.id, 'welcome-screen').get();
+    return new WelcomeScreen(this, data);
```

Comment:
Update this.welcomeScreen while you're at it

Reply:
unsure why we would cache data that might be immediately outdated again (as touched on in https://github.com/discordjs/discord.js/pull/5490#issuecomment-819455212)

Reply:
So then why do we have Guild#welcomeScreen?